WORKSPACE

```diff
@@ -18,10 +18,10 @@ load("@rules_proto//proto:repositories.bzl", "rules_proto_dependencies", "rules_
 rules_proto_dependencies()
 rules_proto_toolchains()
 
-load("@build_bazel_rules_nodejs//:index.bzl", "node_repositories", "npm_install")
```

Comment:
could this rule be using `npm ci`?

Reply:
https://github.com/bazelbuild/rules_nodejs/blob/stable/internal/npm_install/npm_install.bzl#L211 - they don't have it, but I guess I need to investigate if it can be done better there.